From stern@rowland.harvard.edu Thu Feb  1 13:10:04 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Thu, 1 Feb 2007 16:09:59 -0500 (EST)
Subject: EHCI: add debugging message to ehci_bus_suspend
To: Greg KH <greg@kroah.com>
Cc: David Brownell <david-b@pacbell.net>
Message-ID: <Pine.LNX.4.44L0.0702011608420.2566-100000@iolanthe.rowland.org>


This patch (as848) adds a useful little debugging message to let us
know when ehci-hcd's bus_suspend method runs.  The other HCDs have
similar messages; now ehci-hcd doesn't need to feel left out.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-hub.c |    2 ++
 1 file changed, 2 insertions(+)

--- gregkh-2.6.orig/drivers/usb/host/ehci-hub.c
+++ gregkh-2.6/drivers/usb/host/ehci-hub.c
@@ -36,6 +36,8 @@ static int ehci_bus_suspend (struct usb_
 	int			port;
 	int			mask;
 
+	ehci_dbg(ehci, "suspend root hub\n");
+
 	if (time_before (jiffies, ehci->next_statechange))
 		msleep(5);
 
